tap_square/streams.py

```diff
@@ -253,17 +260,19 @@ class Inventories(FullTableStream):
     replication_key = None
 
     def get_pages(self, bookmarked_cursor, start_time):
-        for page, cursor in self.client.get_inventories(start_time, bookmarked_cursor):
-            yield page, cursor
+        yield from self.client.get_inventories(start_time, bookmarked_cursor)
 
 
-class Shifts(Stream):
+class Shifts(FullTableStream):
```

Comment:
I think we should update the metadata in tests/base.py to reflect the change from incremental to full table. I think this should cause the bookmarks to fail if it isn't correct.

Reply:
It definitely didn't change to be full table. It might have just made sense to reuse the code in the base FullTableStream class. But I'll rename it if that's the case.

Reply:
Renamed to BookmarkCursorStream

Reply:
Now that I'm going through the code this name doesn't make sense either, not all of the classes that implement this actually use the bookmark cursor, I'm going to revert it to full table stream, shifts is the only exception, it's not really a full table stream, though in the api code we treat it just like a full table stream, so not sure... maybe call it ClearBookmarkCursorOnExceptionStream, but that's way too implementation specific.

I'm going to revert it to FullTableStream for now, again, if you look in shifts it says the replication method is INCREMENTAL, not FULL, so I think that's more definitive